### PR TITLE
Explicit logging for updates

### PR DESCRIPTION
**Pull request by taion**
This is untested.

### CODE REVIEW

**Comment by taion** [comment]
As in I haven't even tested this manually.

File: src/AuthorizedSocketConnection.js [hunk 1]
```diff
@@ -150,9 +162,9 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
 
     const subscription: AsyncIterable<ExecutionResult> = (result: any);
     for await (const payload of subscription) {
-      this.log('debug', 'processing subscription update', {
-        id,
-        payload,
+      this.log('info', 'emit', {
+        messagePayload: payload,
```
Comment (taion) [inline src/AuthorizedSocketConnection.js]: @yodaiken why is the key `messagePayload`?

File: src/AuthorizedSocketConnection.js [hunk 2]
```diff
@@ -61,11 +61,23 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
...
-      this.config.hasPermission(data, this.getCredentials())
+  isAuthorized = (data: any) => {
+    const isAuthenticated = this.config.credentialsManager.isAuthenticated();
+    const credentials = this.getCredentials();
+
+    const isAuthorized = !!(
+      isAuthenticated && this.config.hasPermission(data, credentials)
     );
+    if (!isAuthorized) {
+      this.log('info', 'unauthorized', {
```
Comment (jquense) [inline src/AuthorizedSocketConnection.js]: I was somewhat reticent to log out credentials but I guess meh?

Comment (taion) [reply src/AuthorizedSocketConnection.js]: The credentials are just the dictionary, right? Not the actual token.

File: src/AuthorizedSocketConnection.js (hunk 1 above)
Comment (yodaiken) [reply src/AuthorizedSocketConnection.js]: feel free to change to payload

File: src/AuthorizedSocketConnection.js (hunk 2 above)
Comment (jquense) [reply src/AuthorizedSocketConnection.js]: Ah right.

**Comment by taion** [comment]
@yodaiken I don't have a good test bench to actually test this code. Do you want to take this over, ensure that it works, and merge when ready?

**Comment by jquense** [comment]
fix the commit message as well when you merge

**Pull request merged**